tests/io/test_data_catalog.py

```diff
@@ -652,3 +652,14 @@ def test_replacing_nonword_characters(self):
         assert "ds2_spark" in catalog.datasets.__dict__
         assert "ds3__csv" in catalog.datasets.__dict__
         assert "jalapeño" in catalog.datasets.__dict__
+
+    def test_no_version_cloud(self):
+        """Check the error if no versions are available for load from cloud storage"""
+        version = Version(load=None, save=None)
+        ds = CSVDataSet("s3://bucket/file.csv", version=version)
```

Comment:
```suggestion
        versioned_dataset = CSVDataSet("s3://bucket/file.csv", version=version)
```

Reply:
Made the change in 211c449b68a52d3204dcb884b9af671d24753b1f